src/odb.c

```diff
@@ -110,16 +125,16 @@ int git_odb__hashobj(git_oid *id, git_rawobj *obj)
 		return -1;
 	}
 
-	hdrlen = git_odb__format_object_header(header, sizeof(header), obj->len, obj->type);
+	if ((error = git_odb__format_object_header(&hdrlen,
+		header, sizeof(header), obj->len, obj->type)) < 0)
+		return error;
```

Comment:
I think the indentation for all these line-broken if statements is a bit weird. I have no idea if that's because of how GitHub displays them, or if you really use all-tabs for the broken line and the first statement. Thing is, you can't easily discern the broken line from the first statement by indentation, which makes those statements harder to read than necessary.

I think in some places, we just indent the broken line by 4 spaces, instead of by tab.